Doc/c-api/datetime.rst

```diff
@@ -141,7 +184,7 @@ Macros to create objects:
    .. versionadded:: 3.7
 
 
-.. c:function:: PyObject* PyTimeZone_FromOffsetAndName(PyDateTime_DeltaType* offset, PyUnicode* name)
+.. c:function:: PyObject* PyTimeZone_FromOffsetAndName(PyDateTime_DeltaType *offset, PyObject *name)
```

Comment:
The first argument type is ``PyObject*``, no?

PyDateTime_CAPI:

```
PyObject *(*TimeZone_FromTimeZone)(PyObject *offset, PyObject *name);
```

Same remark for PyTimeZone_FromOffset().

Reply:
Good catch.